website/addons/dataverse/static/dataverseFangornConfig.js

```diff
@@ -29,29 +29,55 @@ var _dataverseItemButtons = {
             tb.dropzone.hiddenFileInput.click();
             tb.dropzoneItemCache = item;
         }
-        function dataversePublish(event, item, col) {
+        // Get the external Dataverse host before publishing
+        function getDataverseHost(event, item, col) {
+            var host;
+            $.ajax({
```

Comment:
@samanehsan I tried doing this. Unless I'm missing something, though, none of the information stored in item.data actually references the host. For example, here is item.data for a dataset on apitest.dataverse.org:

![screen shot 2016-04-25 at 2 40 38 pm](https://cloud.githubusercontent.com/assets/11521966/14794851/d577697c-0af3-11e6-9dad-6b63b6f6b042.png)